dev/container-tests/arch.yml

```diff
@@ -9,16 +9,12 @@ commandTests:
     command: g++
```

Comment:
We don't run these container tests in CI. It would be nice to add them in a pull request.